From 91d76ea049fd11808030fb3bd18b97b21ff664f0 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Tue, 28 Mar 2023 15:34:45 +0200
Subject: [PATCH] arm64: defconfig: Enable the STM32MP25 ComboPHY

STM32 ComboPHY to generate the PCIe or USB3 refClk.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I619206e155bcd88a9513ccaeb8ce90e36cf5ea52
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/361772
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index c05f7a2e0c7d..45475b1ecd06 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -41,6 +41,7 @@ CONFIG_NVMEM_REBOOT_MODE=y
 CONFIG_NVMEM_STM32_ROMEM=y
 CONFIG_NVMEM_STM32_TAMP=y
 CONFIG_OF_OVERLAY=y
+CONFIG_PHY_STM32_COMBOPHY=y
 CONFIG_PHY_STM32_USB2PHY=y
 CONFIG_PWM_STM32=m
 CONFIG_PWM_STM32_LP=m
-- 
2.39.5

